### DISCUSSION TITLE
2.1.0a questions and comments for tititopher68

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1931] [General Discussion]
First off, damn. I really like the progress. It's now on all 3 of my printers and I forwarded it to several friends. I have a few questions if you could. I didn't see a discussion tab on https://github.com/tititopher68-dev2/Marlin/tree/new_feature so I'd figured I'd post here.

1.)Firmware retraction. I use nozzle wipe and from my understanding wipe isn't supported yet with firmware retraction. I want to continue using slicer retraction. Will firmware retraction effect my print if I don't have it enabled in the slicer?

2.) When the printer homes to start a print, it'll do X0 & Y0, move to X115 Y115 and home Z, then return to the left of the bed to place the pre-extrusion line. Is there a way I can make all my homing at XYZ 0 and not go to the center of the bed?

**Comment by Miq1** [comment]
> 2.) When the printer homes to start a print, it'll do X0 & Y0, move to X115 Y115 and home Z, then return to the left of the bed to place the pre-extrusion line. Is there a way I can make all my homing at XYZ 0 and not go to the center of the bed?

You can comment out the line ``#define Z_SAFE_HOMING`` in ``Configuration.h``. That is what I did to get back to the XYZ=0 home position.

**Comment by alphatrianguli** [comment]
Just compiled and installed the newest firmware - very neat and all looking good so far.

One question for @tititopher68-dev : The box around the Z offset value is blinking - what does this mean?

**Reply by alphatrianguli** [reply]
I noticed, the square keeps blinking throughout the whole print - did I do something wrong?

**Comment by alphatrianguli** [comment]
@tititopher68-dev I noticed another bug: When I load the filament through the "Change filament" menu, even after the loading is done, the status text stays on "Loading..."... I assume this is a bug?
Many thanks for looking at it!

**Reply by alphatrianguli** [reply]
@tititopher68-dev Also, the above problem I mentioned (May 2nd) with the Z offset box blinking is not resolved in the latest version, it blinks again through the print... Just tell me, when you need any more info on this. Many thanks!

**Reply by alphatrianguli** [reply]
@tititopher68-dev
Interesting - I am using the following commands in my startGcode:

G28 ; Home all axes
G29 A ; Activate UBL
G29 L0; Load last used UBL mesh
G29 J ; 3 point probe to tilt the mesh

I use the G29 rather than the M420 S1 Z2 because I read somewhere that M420 and G29 are identical.

Can you post your relevant snippet of start code? Which one of the 3 G29 would I replace with the M420 to test?

Many thanks

**Reply by alphatrianguli** [reply]
@tititopher68-dev Thanks for the hint, will check it out!

Do you also have an idea, why my printer shows "Loading..." long after the filament is completely loaded?

**Comment by Codazed** [comment]
@tititopher68-dev I tried several different BL Touch versions of the 4.2.2 firmware (including one 4.2.7 version just as a sanity check) and none of the versions I tried had the BL Touch in a working state. Probe commands were unrecognized by the firmware and Z homing was non-functional. I built from commit e5e9b3844c714ceab00415b51aeaadb4162b7c5b on branch "new_feature". I'm not sure if I'm doing something wrong.
